justfile

```diff
@@ -1,145 +1,149 @@
 deploy:
     -git pull
     @just pull
     @just up
 
-# Run pre-commit on all files
-lint hook="":
-    pre-commit run {{ hook }} --all-files
-
-# Load any dependencies necessary for actions on the stack without running the webserver
-_deps:
-    @just up "postgres s3 load_to_s3"
-
-# Mount the tests directory and run a particular command
-@_mount-tests command: _deps
-    # The test directory is mounted into the container only during testing
-    @just _dc run \
-        -e AIRFLOW_VAR_INGESTION_LIMIT=1000000 \
-        -v {{ justfile_directory() }}/tests:/opt/airflow/tests/ \
-        -v {{ justfile_directory() }}/pytest.ini:/opt/airflow/pytest.ini \
-        -v {{ justfile_directory() }}/docker:/opt/airflow/docker/ \
-        --rm \
-        {{ SERVICE }} \
-        {{ command }}
-
-# Run a container that can be used for repeated interactive testing
-test-session:
-    @just _mount-tests bash
-
-# Run pytest using the webserver image
-test *pytestargs:
-    @just _mount-tests "bash -c \'pytest {{ pytestargs }}\'"
-
-# Open a shell into the webserver container
-shell user="airflow": up
-    @just _dc exec -u {{ user }} {{ SERVICE }} /bin/bash
-
-# Launch an IPython REPL using the webserver image
-ipython: _deps
-    @just _dc run \
-        --rm \
-        -w /opt/airflow/openverse_catalog/dags \
-        {{ SERVICE }} \
-        bash -c \'ipython\'
-
-# Run a given command in bash using the scheduler image
-run *args: _deps
-    @just _dc run --rm -u {{ DC_USER }} {{ SERVICE }} bash -c \'{{ args }}\'
-
-# Launch a pgcli shell on the postgres container (defaults to openledger) use "airflow" for airflow metastore
-db-shell args="openledger": up
-    @just _dc exec postgres pgcli {{ args }}
-
-# Generate the DAG documentation
-generate-dag-docs fail_on_diff="false":
-    #!/bin/bash
-    set -e
-    DC_USER=root just run 'python openverse_catalog/utilities/dag_doc_gen/dag_doc_generation.py \&\& chmod 666 /opt/airflow/openverse_catalog/utilities/dag_doc_gen/DAGs.md'
-    # Move the file to the top level, since that level is not mounted into the container
-    mv openverse_catalog/utilities/dag_doc_gen/DAGs.md DAGs.md
-    echo -n "Running linting..."
-    # Linting step afterwards is necessary since the generated output differs greatly from what prettier expects
-    just lint &>/dev/null || true
-    echo "Done!"
-    if {{ fail_on_diff }}; then
-      set +e
-      git diff --exit-code DAGs.md
-      if [ $? -ne 0 ]; then
-          printf "\n\n\e[31m!! Changes found in DAG documentation, please run 'just generate-dag-docs' locally and commit difference !!\n\n"
-          exit 1
-      fi
-    fi
-
 # Generate files for a new provider
 add-provider provider_name endpoint +media_types="image":
-    python3 openverse_catalog/templates/create_provider_ingester.py "{{ provider_name }}" "{{ endpoint }}" -m {{ media_types }}
+    python3 catalog/templates/create_provider_ingester.py "{{ provider_name }}" "{{ endpoint }}" -m {{ media_types }}
```

Comment:
Additionally, since the tests themselves have moved, I think this script will need to be updated to generate files in `catalog/tests` rather than `tests/`.

Reply:
👍 This should be fixed by updating [this line](https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/templates/create_provider_ingester.py#L92) in `create_provider_ingester.py`:

```
tests_path = project_path / "tests"
```